core/common/assets/js/api/core/data.js

```diff
@@ -189,27 +195,30 @@ export default class Data extends Commands {
 	 * If the command have query convert it to args.
 	 *
 	 * @param {string} command
-	 * @param {object} argsTarget
+	 * @param {object} args
 	 *
-	 * @returns {string} command
+	 * @returns {ExtractedCommand} command
 	 */
-	commandExtractArgs( command, argsTarget ) {
+	commandExtractArgs( command, args = {} ) {
```

Comment:
Why do you need the args parameter at all? It's not related to this method purpose

Reply:
It used in cases, you want merge, args.